Data archiving workflow instructions editing

ottumm opened the pull request:
- link to data archiving instructions from main README
- minor copy-editing

ottumm committed:
Minor copy editing

ottumm committed:
Link to data archiving (harvesting) instructions

ottumm commented:
Open question: should I add instructions around finding a checker (via e.g. the #epa channel) and doing the checking to the bottom of `harvesting.md`, or will the pipeline app handle that flow?

ottumm commented:
Spoke with @suchthis and it sounds like we're not doing any checking at the event, which simplifies everything nicely!

suchthis commented on README.md:
```diff
@@ -24,6 +24,7 @@
   * Join the #data-archiving slack channel
   * DM a guide your email address for an invite to the pipeline app
   * Please use the same username for both slack and app ("firstnamelastname" or whatever you are comfortable with having online)
+  * Follow the [data archiving instructions](./workflow/harvesting.md)
```
Did you want to link specifically to the harvesting workflow here? Seems like people in this track need to follow both/either the research workflow and the harvesting workflow, and the research comes first.

ottumm replied:
Ah good call.

suchthis commented on workflow/harvesting.md:
```diff
@@ -1,17 +1,17 @@
-# Harvesting
+# Data Archiving (aka "Harvesting")
```
Do you think we should stick with using "data archiving" to mean the track broadly (research & harvest) or should we align it specifically with the harvesting part?

ottumm replied:
Yep, you're right. "Data Archiving" is the name of the track and "Harvesting" is the name of the activity. I'll change it back.

ottumm committed:
Link to both research and harvesting instructions

ottumm committed:
Change name back to “Harvesting”

“Data Archiving” is the name of the track, which includes both research and harvesting.

ottumm committed:
Include #data-archiving-research channel

ottumm committed:
Rename channel #data-archiving-research to #data-research
